### PR TITLE
[Sonic Generations] Terrain and Collision importing tutorials

### PR DESCRIPTION
**Pull request by TheExentist151**
Added tutorials for Sonic Generations and fixed 1 typo in Terrain Import tutorial.

### CODE REVIEW

File: docs/guides/hedgehog-engine/blueblur/levels/importing-collision/importing-collision.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+description: Importing Collision tutorial for Sonic Generations
+---
+!!! info
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+	- [Blender (You can use any other 3D editor)](https://blender.org){ target="_blank"}
```
Comment (PTKay) [inline docs/guides/hedgehog-engine/blueblur/levels/importing-collision/importing-collision.md]: This isn't correctly indented. Sonic GLvl and Blender need to be at the same indentation level

File: docs/guides/hedgehog-engine/blueblur/levels/importing-terrain/importing-terrain.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+description: Importing Terrain tutorial for Sonic Generations
+---
+!!! info
+    This guide uses the following tools:
+
+    - [Sonic GLvl](/tools/hedgehog-engine/blueblur/levels/){ target="_blank"}
+	- [Blender (You can use any other 3D editor)](https://www.blender.org/){ target="_blank"}
```
Comment (PTKay) [inline docs/guides/hedgehog-engine/blueblur/levels/importing-terrain/importing-terrain.md]: This isn't correctly indented. Sonic GLvl and Blender need to be at the same indentation level

File: mkdocs.yml [hunk 3]
```diff
@@ -51,6 +51,8 @@ nav:
             - Animations: guides/hedgehog-engine/blueblur/files/xncp/anims.md
         - Level Editing:
           - Sonic GLvl: guides/hedgehog-engine/blueblur/levels/glvl.md
+          - Importing Terrain: guides/hedgehog-engine/blueblur/levels/importing-terrain/importing-terrain.md
+          - Importing Collision: guides/hedgehog-engine/blueblur/levels/importing-collision/importing-collision.md
```
Comment (PTKay) [inline mkdocs.yml]: You don't need a dedicated folder for these guides. You can place them directly in the `levels` folder, and place their images in the already existing `assets` folder there. The assets however do need their own dedicated folder.

TLDR: Move the guide up one level in the directory tree, and delete their dedicated folders

**Review by PTKay — CHANGES_REQUESTED** [review]
Hey there, thanks for your contribution! I've left some comments so that you can improve the structure a bit, but the content of the guides themselves seem fine overall.

**Comment by PTKay** [comment]
Nice, thanks for the changes! LGTM

**Review by PTKay — APPROVED** [review]
